### PR TITLE
baizhu frames + bugfixes

### PR DESCRIPTION
**Pull request by k0l11**
- fixes baizhu e not ticking more than once in single target
- fixes baizhu q healing on very first tick

### CODE REVIEW

File: internal/characters/baizhu/skill.go [hunk 1]
```diff
@@ -77,13 +78,15 @@ func (c *char) chain(src int, count int) combat.AttackCBFunc {
...
-		})
+		next := c.Core.Combat.RandomEnemyWithinArea(combat.NewCircleHitOnTarget(a.Target, nil, 10), nil)
 		if next == nil {
 			c.skillHealing()
 			return
 		}
+		delay := skillTickInterval
+		if next.Key() == a.Target.Key() {
+			delay += 6 // add some delay in case it's a different target
+		}
```
Comment (skippi) [inline internal/characters/baizhu/skill.go:89]: - The comment here doesn't seem to match what's happening with the `if` check. It looks like we're adding delay if the target stays the same.
- Should we document this 6f delay in the baizhu frame sheet?

File: internal/characters/baizhu/burst.go [hunk 2]
```diff
@@ -12,27 +12,41 @@ import (
...
+	burstRefreshRate  = 146
+	burstShieldExpiry = 152
+	// subject to change
+	burstTickRelease = 21
+	burstTickTravel  = 8
+)
 
 func init() {
 	burstFrames = frames.InitAbilSlice(115)
 	burstFrames[action.ActionSwap] = 114
```
Comment (skippi) [inline internal/characters/baizhu/burst.go:27]: Missing burst cancels here from the sheet.

**Review by skippi — APPROVED** [review]

**Review by skippi — CHANGES_REQUESTED** [review]
Meant for request changes.

File: internal/characters/baizhu/skill.go (hunk 1 above)
Comment (k0l11) [reply internal/characters/baizhu/skill.go]: I wouldn't. All the values here are dependent on distance so I just picked some reasonable value from a quick test in-game

**Review by skippi — APPROVED** [review]

**Pull request merged**